feedback/feedback.py

```diff
@@ -62,12 +43,28 @@ class FeedbackXBlock(XBlock):
     # will default to the ones in default_prompt.
     prompts = List(
         default=[
-            {'freeform': DEFAULT_FREEFORM,
-             'default_text': DEFAULT_DEFAULT,
-             'likert': DEFAULT_LIKERT,
-             'placeholder': DEFAULT_PLACEHOLDER,
-             'scale_text': DEFAULT_SCALETEXT,
-             'icon_set': DEFAULT_ICON}
+            {
+                "freeform": _("What did you learn from this? What was missing?"),
```

Comment:
Why are we removing constants here while Django templates can translate them
Ref: https://docs.djangoproject.com/en/4.2/topics/i18n/translation/#translate-template-tag

Reply:
I was manually adding support to translations, and they didn't work correctly. I tested with moving them to the XBlock runtime. I will revert the changes